ollama/_client.py

```diff
@@ -1152,6 +1207,8 @@ def _parse_host(host: Optional[str]) -> str:
   'https://[0001:002:003:0004::1]:56789/path'
   >>> _parse_host('[0001:002:003:0004::1]:56789/path/')
   'http://[0001:002:003:0004::1]:56789/path'
+  >>> _parse_host('http://host.docker.internal:11434/path')
+  'http://host.docker.internal:11434/path'
```

Comment:
Why the new new test case? This is equivalent to `https://example.com:56789/path` above

Reply:
Oops thanks for the catch - did not mean for this one to be in